.github/workflows/pull-gitleaks.yml

```diff
@@ -0,0 +1,23 @@
+name: pull-gitleaks
```

Comment:
```suggestion
name: gitleaks-detector
```
how about that ?

Reply:
I would either keep it as it is to indicate that this entire file comes for the repository template or align with our naming and make it something like `PR Code checks / gitleaks`

Reply:
`PR Code checks / gitleaks` sounds better

Reply:
Done